### PR TITLE
tools for workflow/atomic action conforms

### PR DESCRIPTION
**Pull request by oborys**

### CODE REVIEW

File: manual-sample-repo/README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ In the event you make use of 3rd party code, it is required by some licenses, an
 8. Add a NOTICE file with copyright if you use GPLv3 or Apache 2.0 license ([sample NOTICE file](https://github.com/CiscoDevNet/opendaylight-sample-apps/blob/master/NOTICE)).
 9. Dockerize app or part of an app like server/client.
 10. At the top of the `Readme.md` file add a hash symbol and the full use case name to create a useful Readme title. As an example, write `# Devicebanner, updates the banner motd on a network device` instead of just 'devicebanner'. 
+11. If your repo is connected with Cisco SecureX orchestration workflow, please check if your workflow or atomic action conforms to their best practices using [this tool](https://ciscosecurity.github.io/sxo-05-security-workflows/analyzer/)
```
Comment (annegentle) [inline manual-sample-repo/README.md:120]: Add a period to the end of this line and it's good to go! Nice additionl

**Review by annegentle — CHANGES_REQUESTED** [review]
One small request to add punctuation - good update.

**Comment by oborys** [comment]
Thanks, I've add punctuation

**Review by annegentle — APPROVED** [review]
Looks good!

**Pull request merged**